demisto_sdk/common/rest_api.py

```diff
@@ -0,0 +1,192 @@
+import os
+import json
+from urllib.parse import urljoin
+
+import requests
+
+from demisto_sdk.common.tools import print_v
+
+
+DEMISTO_API_KEY_ENV = 'DEMISTO_API_KEY'
+INTEGRATION_UPLOAD_PATH = "/settings/integration-conf/upload"
+ENTRY_EXECUTE_PATH = "/entry/execute/sync"
+ENTRY_DOWNLOAD_PATH = "/entry/download/"
+INVESTIGATION_SEARCH_PATH = "/investigations/search"
+
+
+class Client:
+    """Demisto REST API Client
+    
+    Args:
+        base_url (str): URL of Demisto instance
+        verify_cert (bool, optional): Verify remote certificate. Defaults to True.
+        verbose (bool, optional): Verbose logging.
+    """
+    def __init__(self, base_url: str, verify_cert: bool = True, verbose: bool = False):
+        self.base_url = base_url
+        self.verify_cert = verify_cert
+        self.log_verbose = verbose
+
+    def get_api_key(self):
+        """Retrieve the API Key
+        
+        Raises:
+            RuntimeError: if the API Key environment variable is not found
+        
+        Returns:
+            str: API Key
+        """
+        ans = os.environ.get(DEMISTO_API_KEY_ENV, None)
```

Comment:
what do you think about adding the option to pass this argument using the argparse as well?

Reply:
From a security point of view is not the best as it will show up in the running processes. But it would improve usability. What options should I use?

Reply:
@glicht what do you think about this?